console

```diff
@@ -74,6 +74,7 @@ if __name__ == "__main__":
 
         if not os.path.exists(backend.META) or not os.path.isdir(backend.META):
             backend.clone()
+            backend.pull()
         else:
             backend.pull()
```

Comment:
If we are doing a pull in both cases let's just move It out.

```python
        if not os.path.exists(backend.META) or not os.path.isdir(backend.META):
            backend.clone()

        backend.pull()
```

Reply:
Yes -- we need to do the pull in both cases, or we don't get the branch, if the user has configured a different branch from main/master.